### PR TITLE
Add undo/redo

### PR DESCRIPTION
**Pull request by nicholette**
Add undo/redo functionality

To be merged after #2570

### CODE REVIEW

File: client/app/bundles/course/assessment/submission/translations.js [hunk 1]
```diff
@@ -342,6 +342,13 @@ export const scribingTranslations = defineMessages({
   layers: {
     id: 'course.assessment.submission.answer.scribing.layers',
     defaultMessage: 'Layers',
```
Comment (kxmbrian) [inline client/app/bundles/course/assessment/submission/translations.js]: Missing `},`

Comment (nicholette) [reply client/app/bundles/course/assessment/submission/translations.js]: Oops, fixed that.

**Comment by kxmbrian** [comment]
Could you rebase?

**Comment by nicholette** [comment]
@kxmbrian Rebased, made some changes too

File: client/app/bundles/course/assessment/submission/components/ScribingView/ScribingCanvas.jsx [hunk 2]
```diff
@@ -631,6 +649,61 @@ export default class ScribingCanvas extends React.Component {
...
+      this.canvas.clear();
+      this.canvas.setBackground();
+      this.props.scribing.layers.forEach(layer => this.canvas.add(layer.scribbleGroup));
+      userScribbles.forEach(scribble => this.canvas.add(scribble));
+      this.canvas.renderAll();
+      this.isScribblesLoaded = true;
+
+      this.props.setCurrentStateIndex(this.props.answerId, this.props.scribing.currentStateIndex + 1);
+    }
+  }
```
Comment (kxmbrian) [inline client/app/bundles/course/assessment/submission/components/ScribingView/ScribingCanvas.jsx]: Could you DRY up `undo` and `redo`?

Comment (nicholette) [reply client/app/bundles/course/assessment/submission/components/ScribingView/ScribingCanvas.jsx]: oops >< sure!

**Pull request merged**